code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -1,285 +1,646 @@
+	icon_state = "clockwork_treads"
+	armor = list("melee" = 15, "bullet" = 15, "laser" = 10, "energy" = 10, "bomb" = 50, "bio" = 10, "rad" = 0, "fire" = 70, "acid" = 50)
+
+/obj/item/clothing/glasses/hud/clock
+	name = "Malfunctioning Judicial Visor"
+	desc = "An ancient visor. Looks like it's malfunctioning, but it still functions to see those unworthy."
+	worn_icon = 'icons/Fulpicons/clockchap.dmi'
+	icon ='icons/obj/clothing/clockwork_garb.dmi'
+	icon_state = "judicial_visor_1"
+	vision_flags = SEE_MOBS
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+
+/obj/item/clothing/suit/space/clockwork // Clockwork suits were caracterized by their extreme brute and bullet resistance, yet still weak to lasers.
+	inhand_icon_state = "clockwork_cuirass"
+	worn_icon = 'icons/Fulpicons/clockchap.dmi'
+	icon = 'icons/obj/clothing/clockwork_garb.dmi'
+	icon_state = "clockwork_cuirass"
+	name = "Clockwork Cuirass"
+	desc = "Ancient suit made out of brass. It seems to provide robust protection against most damage types. It glows with a golden, ancient and forgotten energy."
+	armor = list("melee" = 50, "bullet" = 50, "laser" = 20, "energy" = 20, "bomb" = 30, "bio" = 100, "rad" = 30, "fire" = 100, "acid" = 100)
+	slowdown = 0
+	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS // The martial art is literally based around fire JFC do you want to burn?
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	actions_types = list(/datum/action/item_action/toggle_spacesuit, /datum/action/item_action/initialize_clock_suit)
+
+	//Using ninja's suit code to make the Cuirass lock onto you
+	var/mob/living/carbon/human/affecting = null
+	var/s_initialized = 0
+	var/s_busy = FALSE
+	var/s_delay = 40
+	var/obj/item/clothing/shoes/combat/swat/bronze/c_shoes
+	var/obj/item/clothing/head/helmet/space/clockwork/c_helmet
+	var/obj/item/clothing/gloves/combat/clockwork/c_gauntlets
+	var/obj/item/clothing/glasses/hud/clock/c_visor
+
+/datum/action/item_action/initialize_clock_suit
+	name = "Activate Cuirass"
+
+#define CLOCK_SUIT_TRAIT "clockwork-cuirass"
+
+// Initiailizing it gives you NODROP
+
+/obj/item/clothing/suit/space/clockwork/proc/lock_suit(mob/living/carbon/human/H)
+	if(!istype(H))
+		return FALSE
+	if(!istype(H.head, /obj/item/clothing/head/helmet/space/clockwork))
+		to_chat(H, "<span class='userdanger'/span> The Justiciar requires you to have your helmet to unlock your potential.")
+		return FALSE
+	if(!istype(H.shoes, /obj/item/clothing/shoes/combat/swat/bronze))
+		to_chat(H, "<span class='userdanger'/span> The clockwork Justiciar requires you to wear his treads.")
+		return FALSE
+	if(!istype(H.gloves, /obj/item/clothing/gloves/combat/clockwork))
+		to_chat(H, "<span class='userdanger'/span> Brass gauntlets are essential to enable your power.")
+		return FALSE
+	if(!istype(H.glasses, /obj/item/clothing/glasses/hud/clock))
+		to_chat(H, "<span class='userdanger'/span> Unlock your True Sight through my visor before activating this.")
```

Comment:
wait lmao you can't even choose to NOT use the glasses, you HAVE to get welder eyes or a welding gas mask if you want flash protection

holy shit this entire kit gets dunked on by default sec gear lmao